concourse/pipelines/create-cloudfoundry.yml

```diff
@@ -2115,6 +2123,96 @@ jobs:
                   paas-billing-collector \
                   -f manifest-collector.yml
 
+      - task: deploy-noisy-neighbor-nozzle
+        config:
+          platform: linux
+          image_resource: *cf-cli-image-resource
+          params:
+            AWS_REGION: ((aws_region))
+            SYSTEM_DNS_ZONE_NAME: ((system_dns_zone_name))
+            DEPLOY_ENV: ((deploy_env))
+            UAA_ENDPOINT: "https://uaa.((system_dns_zone_name))"
+            LOGGREGATOR_ENDPOINT: "wss://doppler.((system_dns_zone_name)):443"
+            APPS_DNS_ZONE_NAME: ((apps_dns_zone_name))
+          inputs:
+            - name: paas-cf
+            - name: gh-release-noisy-neighbor-nozzle
+            - name: config
+            - name: cf-vars-store
+          run:
+            path: sh
+            args:
+              - -e
+              - -u
+              - -c
+              - |
+
+                VAL_FROM_YAML=$(pwd)/paas-cf/concourse/scripts/val_from_yaml.rb
+                CF_CLIENT_SECRET=$($VAL_FROM_YAML secrets_uaa_clients_noisy_neighbor_nozzle_secret cf-vars-store/cf-vars-store.yml)
+
+                . ./config/config.sh
+                echo | cf login -a "${API_ENDPOINT}" -u "${CF_ADMIN}" -p "${CF_PASS}" -o admin -s monitoring
+
+                tar -zxf gh-release-noisy-neighbor-nozzle/noisy-neighbor.tgz -C .
+
+                cd noisy-neighbor
+
+                CLIENT_ID="noisy-neighbor-nozzle"
+                CLIENT_SECRET="${CF_CLIENT_SECRET}"
+                SKIP_CERT_VERIFY="false"
+                UAA_ADDR="${UAA_ENDPOINT}"
+                LOGGREGATOR_ADDR="${LOGGREGATOR_ENDPOINT}"
+                NOZZLE_INSTANCES="2"
+                SUBSCRIPTION_ID="$(date +%s | sha256sum | base64 | head -c 64 ; echo)"
```

Comment:
This is a bit of an odd way of generating a random subscription id. If you just want 64 characters of random base64 you can `head -c 48 < /dev/random | base64`.

Reply:
Good point